manager/gui/src/components/alerts/LrAlert.vue

```diff
@@ -100,35 +109,94 @@ export default class LrAlert extends Vue {
       this.showDataTable = true;
     }
   }
+
+  getAlertsList(queryFilters?: string) {
+    const filter = queryFilters ? `?${queryFilters}` : "";
+    const endpoint = `alerts/list${filter}`;
+    Api.getData(endpoint, { isDummy: true }).then((resp: any) => {
+      this.alerts = resp["data"];
+    });
+  }
+
+  setSeverityFilter() {
+    if (this.severity) {
+      let filterList: SgtFilterObject[] = [];
+      const advanceForm = [...this.alertConst.searchConfig.advanceForm];
+      const updatedAdvanceForm = advanceForm.map((element) => {
+        if (element.name === "severity") {
+          element.value = this.severity;
+          filterList.push({
+            label: element.label,
+            name: element.name,
+            value: element.value,
+          });
+        }
+        return element;
+      });
+      this.alertConst.searchConfig.advanceForm = updatedAdvanceForm;
+    }
+  }
+
   getColor(item: any) {
     return this.alertConst.severityList[item.severity];
   }
+
   updateRecord(tableDataConfig: SgtDataTableFilterSortPag) {
-    // code for API call
+    let queryFilters = ``;
+    tableDataConfig.filterList.forEach((filter) => {
+      queryFilters += `${filter.name}=${filter.value}`;
+    });
+    this.getAlertsList(queryFilters);
     this.chips = tableDataConfig.filterList;
```

Comment:
chips should be updated only after API returns value

Reply:
Moved the chips assignment after the API call.